sparklingpandas/pcontext.py

```diff
@@ -141,22 +151,57 @@ def csv_rows(partition_number, rows):
 
     def parquetFile(self, *paths):
         """Loads a Parquet file, returning the result as a L{Dataframe}.
+
+        Parameters
+        ----------
+        paths: string, variable length
+             The path(s) of the parquet files to load. Should be Hadoop style
+             paths (e.g. hdfs://..., file://... etc.).
+        Returns
+        -------
+        A L{Dataframe} of the contents of the parquet files.
         """
-        return self.from_spark_rdd(self.sql_ctx.parquetFile(paths),
-                                   self.sql_ctx)
+        return self.from_spark_rdd(self.sql_ctx.parquetFile(paths))
 
     def jsonFile(self, path, schema=None, sampling_ratio=1.0):
         """Loads a text file storing one JSON object per line as a
         L{Dataframe}.
+        Parameters
+        ----------
+        path: string
+             The path of the json files to load. Should be Hadoop style
+             paths (e.g. hdfs://..., file://... etc.).
+        schema: StructType, optional
+             If you know the schema of your input data you can specify it. The
+             schema is specified using Spark SQL's schema information. If not
+             specified will sample the json records to determine the schema.
+        sampling_ratio: int, default=1.0
+             Percentage of the records to sample when infering schema.
+             Defaults to all records for safety, but you may be able to set to
+             a lower ratio if the same fields are present accross records or
+             your input is of sufficient size.
+        Returns
+        -------
+        A L{Dataframe} of the contents of the json files.
         """
         schema_rdd = self.sql_ctx.jsonFile(path, schema, sampling_ratio)
-        return self.from_spark_rdd(schema_rdd, self.sql_ctx)
+        return self.from_spark_rdd(schema_rdd)
 
     def from_pd_data_frame(self, local_df):
-        """Make a distributed dataframe from a local dataframe. The intend use
-        is for testing. Note: dtypes are re-infered, so they may not match."""
+        """Make a Sparkling Panda's dataframe from a local Panda's dataframe.
```

Comment:
No apostrophe, (Pandas not Panda) and capitalize dataframe, maybe even link to dataframe docs?